### PR TITLE
Add docker-compose.yml file

### PR DESCRIPTION
**Pull request by shashidesai**
* adding docker-compose for easier setup.

### CODE REVIEW

**Comment by mefellows** [comment]
OK, there are two problems with your compose file:

1) You are setting `PACT_BROKER_DATABASE_HOST` to `localhost` - you can't do this, the Ruby app (container) will try to connect to itself on port 5432 which is not where the DB is running. It's going to be running in the postgres container, which will be given the hostname `postgres` (the name of the `link`)
2) You have the wrong environment variable for the username. It should be `PACT_BROKER_DATABASE_USERNAME` but you are using `PACT_BROKER_USERNAME`.

If you fix both of those things, your compose file should work and we'll accept this PR.

Also, please squash your commits.

**Comment by shashidesai** [comment]
I corrected the file as per comments. it works fine now.
And I have never done this -'Also, please squash your commits.' will check on how to do that.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ This repository deploys [Pact Broker](https://github.com/bethesque/pact_broker)
 * As the native dependencies for a postgres driver are baked into the docker container, you are limited to using postgres as a database.
 * Apart from creating a postgres database no further preparation is required.
 
+## Alternative setup using docker-compose
+
+1. Use the docker-compose.yml file from the repository, modify and use as needed. 
+2. Use `docker-compose up` to get pack broker running.
```
Comment (mefellows) [inline README.md]: "pack broker" = "Pact Broker"

**Comment by mefellows** [comment]
I can squash from here actually, just fix up the spelling mistake and i'll bring in. Thanks!

**Comment by shashidesai** [comment]
thanks @mefellows , you are fast !

File: README.md (hunk 1 above)
Comment (shashidesai) [inline README.md]: fixed

**Comment by shashidesai** [comment]
maybe small but its my first contribution to a repo outside of work !

**Comment by mefellows** [comment]
Thanks @shashidesai, hopefully this is the start of many!

**Pull request merged**